src/utilities/core-utils.test.ts

```diff
@@ -91,6 +91,126 @@ describe("CoreUtils", () => {
 
     // #endregion getRandomEnum
 
+    // -----------------------------------------------------------------------------------------
+    // #region numericEnumToPojo
+    // -----------------------------------------------------------------------------------------
+
+    describe("numericEnumToPojo ", () => {
+        test("when given numeric enum, returns an object of values", () => {
+            // Arrange
+            enum testEnum {
+                first,
+                second,
+                third,
+            }
+
+            // Act
+            const result = CoreUtils.numericEnumToPojo(testEnum);
+            const values = CoreUtils.objectToArray(result);
+
+            // Assert
+            expect(values).toContain(testEnum.first);
+            expect(values).toContain(testEnum.second);
+            expect(values).toContain(testEnum.third);
+            expect(values).toHaveLength(3);
+        });
+
+        test("when given non numeric enum, returns object of values", () => {
+            // Arrange
+            enum testEnum {
+                first = "First", second = "Second",
```

Comment:
nitpick: each element should be on it's own line

Reply:
I copied from another function. Will fix them all